Code/Framework/AzFramework/AzFramework/ProjectManager/ProjectManager.cpp

```diff
@@ -105,7 +105,14 @@ namespace AzFramework::ProjectManager
             }
 
             AzFramework::ProcessLauncher::ProcessLaunchInfo processLaunchInfo;
-            processLaunchInfo.m_commandlineParameters = executablePath.String() + commandLineArgs;
+
+            AZStd::vector<AZStd::string> launchCmd = {executablePath.String()};
+            for(auto& commandLineArg : commandLineArgs)
```

Comment:
nitpick: missing space between `for(`

Reply:
Actually a for loop isn't needed here, you can use the `AZStd::vector::insert` to just copy all of the commandLineArgs

```
AZStd::vector<AZStd::string> launchCmd = { executablePath.String() };
launchCmd.insert(launchCmd.end(), commandLineArgs.begin(), commandLineArgs.end());
processLaunchInfo.m_commandlineParameters = AZStd::move(launchCmd);
```